official/recommendation/ncf_keras_benchmark.py

```diff
@@ -213,3 +213,50 @@ def benchmark_2_gpus(self):
     self._setup()
     FLAGS.num_gpus = 2
     self._run_and_report_benchmark()
+
+class NCFKerasPerf(NCFKerasBenchmarkBase):
+  """Benchmark NCF model performance  using real data."""
+
+  def __init__(self,
+               output_dir=None,
+               root_data_dir=None,
+               default_flags=None,
+               **kwargs):
+
+    default_flags = {}
+    default_flags['dataset'] = 'ml-20m'
+    default_flags['num_gpus'] = 1
+    default_flags['train_epochs'] = 17
+    default_flags['clean'] = True
+    default_flags['batch_size'] = 1048576
+    default_flags['learning_rate'] = 0.0045
+    default_flags['beta1'] = 0.25
+    default_flags['beta2'] = 0.5
+    default_flags['epsilon'] = 1e-8
+    default_flags['layers'] = [256, 256, 128, 64]
+    default_flags['num_factors'] = 64
+    default_flags['hr_threshold'] = 0.635
+    default_flags['ml_perf'] = True
+    default_flags['use_synthetic_data'] = False
+    default_flags['data_dir'] = os.path.join(root_data_dir, NCF_DATA_DIR_NAME)
+
+    super(NCFKerasPerf, self).__init__(
+        output_dir=output_dir,
+        default_flags=default_flags,
+        **kwargs)
+
+  def _extract_benchmark_report_extras(self, stats):
+    metrics = []
+    metrics.append({'name': 'exp_per_second',
+                    'value': stats['avg_exp_per_second']})
+    return metrics
+
+  def benchmark_1_gpu_ctl_perf(self):
+    self._setup()
+    FLAGS.keras_use_ctl = True
+    self._run_and_report_benchmark()
+
+  def benchmark_1_gpu_perf(self):
```

Comment:
r/benchmark_1_gpu_perf/benchmark_1_gpu_mlperf_like

Add a comment about being mlperf like.  I did something similar in ResNet that I thought was a nice reminder to anyone that sees it as to what the test is doing and why.  
https://github.com/tensorflow/models/blob/master/official/resnet/keras/keras_imagenet_benchmark.py#L120